diff --git a/setup.py b/setup.py
index 69f5b62..3df6a35 100644
--- a/setup.py
+++ b/setup.py
@@ -10,12 +10,12 @@
 Ref: https://github.com/pypa/sampleproject/blob/master/setup.py
 """
 
+import os
 from codecs import open as c_open
 from os.path import abspath, dirname, join
 
 from setuptools import setup
 
-
 CWD = abspath(dirname(__file__))
 
 # Get bidict's package metadata from ./bidict/metadata.py.
@@ -43,9 +43,7 @@ with c_open(join(CWD, 'README.rst'), encoding='utf-8') as f:
 
 # Manually keep these version pins in sync with those in .travis.yml and .pre-commit-config.yaml.
 
-SETUP_REQS = [
-    'setuptools_scm',
-]
+SETUP_REQS = []
 
 SPHINX_REQS = [
     'Sphinx < 3',
@@ -107,13 +105,13 @@ EXTRAS_REQS = dict(
     pylint=PYLINT_REQS,
 )
 
+VERSION = os.environ['PV']
+with open('bidict/_version.py', 'wt') as fv:
+    fv.write("version = u'{}'".format(VERSION))
+
 setup(
     name='bidict',
-    use_scm_version={
-        'version_scheme': 'guess-next-dev',
-        'local_scheme': 'dirty-tag',
-        'write_to': 'bidict/_version.py',
-    },
+    version=VERSION,
     author=METADATA.__author__,
     author_email=METADATA.__email__,
     description=METADATA.__description__,
