diff --git a/mps_youtube/util.py b/mps_youtube/util.py
index da65fcf..12e2eee 100644
--- a/mps_youtube/util.py
+++ b/mps_youtube/util.py
@@ -343,7 +343,9 @@ def real_len(u, alt=False):
 
 def yt_datetime(yt_date_time):
     """ Return a time object, locale formated date string and locale formatted time string. """
-    time_obj = time.strptime(yt_date_time, "%Y-%m-%dT%H:%M:%SZ")
+    d = datetime.fromisoformat(yt_date_time[:-1])
+    y = d.strftime('%Y-%m-%d %H:%M:%S')
+    time_obj = time.strptime(str(y), "%Y-%m-%d %H:%M:%S")
     locale_date = time.strftime("%x", time_obj)
     locale_time = time.strftime("%X", time_obj)
     # strip first two digits of four digit year
@@ -353,7 +355,8 @@ def yt_datetime(yt_date_time):
 
 def yt_datetime_local(yt_date_time):
     """ Return a datetime object, locale converted and formated date string and locale converted and formatted time string. """
-    datetime_obj = datetime.strptime(yt_date_time, "%Y-%m-%dT%H:%M:%SZ")
+    datetime_obj = datetime.fromisoformat(yt_date_time[:-1])
+    # datetime_obj = datetime.strptime(yt_date_time, "%Y-%m-%dT%H:%M:%SZ")
     datetime_obj = utc2local(datetime_obj)
     locale_date = datetime_obj.strftime("%x")
     locale_time = datetime_obj.strftime("%X")
