diff --git a/mps_youtube/players/mpv.py b/mps_youtube/players/mpv.py
index a9d10be..d9f2e0f 100644
--- a/mps_youtube/players/mpv.py
+++ b/mps_youtube/players/mpv.py
@@ -185,7 +185,7 @@ class mpv(CmdPlayer):
                         observe_full = True
 
                     if resp.get('event') == 'property-change' and resp['id'] == 1:
-                        if resp['data'] is not None:
+                        if resp.get('data') is not None:
                             elapsed_s = int(resp['data'])
 
                     elif resp.get('event') == 'property-change' and resp['id'] == 2:
